app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -25,10 +25,13 @@ import scala.util.{Failure, Success, Try}
 
 private[mappings] class LocalDateFormatter(
```

Comment:
you can bring invalidiKey up a line and move the indentation here I think? Like how you have done for populateErrorMsg